### DISCUSSION TITLE
[Data Importer] Taking the default Firefly port 8080 despite setting env variables

### DISCUSSION THREAD

**Discussion by lucasltt** [firefly-iii/firefly-iii#5628] [Firefly III]
### Support guidelines

- [X] I've read the [support guidelines](https://github.com/firefly-iii/firefly-iii/blob/main/.github/support.md)

### I've found a bug and checked that ...

- [X] ... [the documentation](https://docs.firefly-iii.org/) does not mention anything about my problem
- [X] ... there are no open or closed issues that are related to my problem
- [X] ... it's [definitely me, not you](https://github.com/firefly-iii/firefly-iii/blob/main/.github/its_you_not_me.md)

### Description

Hi! I have a local Firefly on docker with a port mapping from 8082 to 8080. I just upgraded to data importer and despite setting the  env variables as mentioned in the documentaiton, authentication "ignores" port 8082 and just foward to port 8080. 

docker run -d \
  --name=fireflyimporter \
  -e FIREFLY_III_CLIENT_ID=76 \
  -e FIREFLY_III_URL=http://10.0.1.154:8082 \
   -p 8081:8080 \
  fireflyiii/data-importer:latest

When I log into data importer "10.1.154:8081" it redirects to

http://10.0.1.154:8080/oauth/authorize?client_id=76&redirect_uri=http%3A%2F%2F10.0.1.154%3A8081%2Fcallback&response_type=code&scope=&state=dTm7qiHgOZrfbZsQX4W0M3Sovq3WJi2uNT0rxDV3&code_challenge=SVxIAJIEZ1tH2voKMqjCyjyF4fLcvF8nRlOAGmlWv1g&code_challenge_method=S256

If I change the url above to 8082 then it autenticates.

Thank you very much

### Expected behaviour

Redirect to port 8082

### Debug information

No info

**Comment by JC5** [comment]
I need some logs. I can't replicate this on the [public data importer](https://data-importer.firefly-iii.org/). If I input a port, it works.

**Comment by lucasltt** [comment]
Hi, please find the docker logs below. One thing that I noted is that I'm setting VANITY_URL=http://10.0.1.154:8082  but on the logs it shows as  http://10.0.1.154:8080

https://pastebin.com/VXGsvdsY

**Comment by JC5** [comment]
Clear your cookies, then try again.
